Gemfile

```diff
@@ -0,0 +1,54 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.6.3'
```

Comment:
it'd be a good idea to leave this version a bit more loose https://andycroll.com/ruby/use-loose-ruby-versioning-in-your-gemfile/